packages/mobile/test/schemas.ts

```diff
@@ -645,6 +645,7 @@ export const v19Schema = {
         usdPrice: '1',
         balance: '10',
         isCoreToken: true,
+        priceFetchedAt: Date.now(),
```

Comment:
We should avoid changing past schemas. The main reason is to make it easier for us to track down how it evolves over time and prevent unexpected issues for existing users.

Reply:
I wasn't sure how to go about adding this, but I just copy/pasted the tokenBalances object and added the extra field, let me know if you think we should do it differently :)